src/main/java/hlf/java/rest/client/service/impl/UpdateChannelImpl.java

```diff
@@ -217,4 +226,21 @@ private FabricNodeOUs.Builder getFabricNodeOUs(
                 .setCertificate(ByteString.copyFrom(nodeCert)))
         .setEnable(true);
   }
+  /**
+   * @param capabilities capabilities need to be added to config
+   * @return channel capabilities
+   */
+  private Configtx.ConfigValue getCapabilities(String... capabilities) {
+    Configtx.ConfigValue.Builder valueBuilder = Configtx.ConfigValue.newBuilder();
+    Configuration.Capabilities.Builder capabilitiesBuilder =
+        Configuration.Capabilities.newBuilder();
+    valueBuilder.setModPolicy(DEFAULT_MOD_POLICY);
+    valueBuilder.setVersion(0);
+    for (String capability : capabilities) {
+      capabilitiesBuilder.putCapabilities(
+              capability, Configuration.Capability.newBuilder().build());
+    }
+    valueBuilder.setValue(capabilitiesBuilder.build().toByteString());
+    return valueBuilder.build();
+  }
```

Comment:
`FabricChannelUtil` is there for all the channel related code that is needed for channel creation and update, move this code and `ChannelServiceImpl` in that class and call that method in this class

Reply:
I note your point regarding moving thi function to FabricChannelUtil, but since the ChannelService class exists, its impl class should also exist and not be moved to a util in the context of this PR.

Reply:
I meant the duplicated code wrt `getCapabilities` in `ChannelServiceImpl` and your PR code, move it to `FabricChannelUtil`